README.md

```diff
@@ -161,6 +164,13 @@ This project uses Firestore and Functions, so we will only need to configure emu
 
 ### Deploying changes
 
-Deploys are handled automatically by the [Firebase functions deploy](https://github.com/covid-projections/act-now-links-service/actions/workflows/functions-deploy.yml) Github action. The workflow is triggered on pushes to the `main` and `develop` branches so that the deployed functions will reflect the most up-to-date changes in `main`/`develop`.
+Deploys are handled automatically by the [Firebase functions deploy](https://github.com/covid-projections/act-now-links-service/actions/workflows/functions-deploy.yml) Github action. The workflow is triggered on pushes to the `main` and `develop` branches so that the deployed functions will reflect the most up-to-date status of the repository.
 
 If need be, you can deploy functions yourself by running `yarn deploy` in `functions/`, but this is generally discouraged as it disrupts the symmetry between Github and Firebase.
+
+
+## Insomnia
+
+Insomnia can be used to test, debug, and store requests. To download and install the Insomnia client see https://insomnia.rest/download. 
+
+To work with the Act Now Links Service API in Insomnia, from the Insomnia dashboard select `Create > Import From > File` and select [`functions/insomnia-config.yaml`](./functions/insomnia-config.yaml).
```

Comment:
Maybe I'm confused, but I don't see "Create" anywhere in the UI:
![image](https://user-images.githubusercontent.com/206364/198076432-6d793ef0-ea01-41f3-8601-94c98c6baace.png)

There's no "Create" menu either. I have Insomnia, Edit, View, Window, Tools, Help.

Reply:
Maybe I installed / used the wrong thing?

Reply:
Ah, thanks for testing--that's the right client and you're right!

I was looking at this screen (below), but the screen you posted is the home/default screen. 
<img width="1424" alt="image" src="https://user-images.githubusercontent.com/55333380/198125237-579692c6-a49f-4793-a556-211a919c8d8d.png">

There's a way to import data through the Insomnia menu, so I've updated this to document that

Reply:
Oh!  I see.  That worked, though TBH it's still a little confusing because it tells me it imported stuff but it still leaves me looking at the empty "New document" that I guess it created by default.  So I'd maybe put the instructions back how they were before but prefix with "Go to the main dashboard screen (e.g. by clicking "Insomnia" at the top of the screen)." or something like that.  Not a big deal though.  If I had played around with the tool a bit more I would have figured out.  It was just confusing / disorienting at first.

Reply:
Ah, interesting. Sounds good! I've reverted the docs and added the prefix 🙏